src/main/java/com/spotify/reaper/resources/ClusterResource.java

```diff
@@ -253,5 +290,10 @@ public Response deleteCluster(
     return Response.serverError().entity("delete failed for schedule with name \""
                                          + clusterName + "\"").build();
   }
+  
+  private Set<String> parseSeedHosts(String seedHost) {
+    return Arrays.stream(seedHost.split(",")).map(seed -> seed.trim()).collect(Collectors.toSet());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1612)